x-pack/filebeat/input/awss3/state.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package awss3
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// State is used to communicate the reading state of a file
+type State struct {
+	Id           string    `json:"id" struct:"id"`
```

Comment:
```suggestion
	ID           string    `json:"id" struct:"id"`
```

Reply:
there is a mage target to apply gofmt, is there?